libs/libcommon/src/libcommon/viewer_utils/rows.py

```diff
@@ -155,7 +157,27 @@ def create_first_rows_response(
         ) from err
 
     # truncate the rows to fit within the restrictions, and prepare them as RowItems
-    columns_to_keep_untruncated = [col for col, feature in features.items() if isinstance(feature, (Image, Audio))]
+    columns_to_keep_untruncated = [
+        col
+        for col, feature in features.items()
+        if isinstance(feature, (Image, Audio))
+        or (  # column of URLs
+            isinstance(feature, Value)
+            and feature.dtype == "string"
+            and len(transformed_rows) > 0
+            and sum(
+                (
+                    col in row
+                    and isinstance(row[col], str)
+                    and row[col].startswith("http://")
+                    or row[col].startswith("https://")
```

Comment:
hmmm is the precedence correct? I would have enclosed the two last lines with parentheses

Reply:
good catch, I'll fix this